test/basics.jl

```diff
@@ -6,13 +6,15 @@ struct CSVString
 end
 
 Base.parse(::Type{CSVString}, s::String) = CSVString(s)
+Base.tryparse(::Type{CSVString}, s::String) = CSVString(s)
```

Comment:
why are these definitions needed?

Reply:
In the fix for https://github.com/JuliaData/Parsers.jl/pull/86 and #860, Parsers now requires `tryparse` to be defined instead of `Base.parse` in the custom type fallback.